common/shlibs

```diff
@@ -149,6 +149,7 @@ libglapi.so.0 libglapi-7.11_1
 libgbm.so.1 libgbm-9.0_1
 libOpenGL.so.0 libglvnd-1.3.0_1
 libGLX.so.0 libglvnd-1.3.0_1
+libEGL.so libglvnd-1.6.0_1
```

Comment:
This isn't really the right fix, we don't want packages linking against unversioned libraries from other packages. `vlc` should be made to link against the versioned library instead probably.

Anyway, I filed a ticket for this: https://github.com/void-linux/void-packages/issues/46974

Reply:
Actually, never mind, this worked for me (also has some misc changes in there, don't mind me):

```diff
diff --git a/srcpkgs/vlc/template b/srcpkgs/vlc/template
index d24428db5b8..baa76782f8e 100644
--- a/srcpkgs/vlc/template
+++ b/srcpkgs/vlc/template
@@ -63,7 +63,7 @@ case "$XBPS_TARGET_MACHINE" in
 		configure_args+=" --disable-altivec"
 		build_options_default+=" opengl"
 		;;
-	armv[67]*)
+	armv[67]*|aarch64*)
 		# XXX only for rpi
 		build_options_default+=" rpi"
 		;;
@@ -76,20 +76,12 @@ fi
 if [ "$build_option_rpi" ]; then
 	configure_args+=" --enable-omxil --enable-omxil-vout --enable-rpi-omxil"
 	makedepends+=" rpi-userland-devel"
-
-	CFLAGS+=" -I${XBPS_CROSS_BASE}/opt/vc/include"
-	CFLAGS+=" -I${XBPS_CROSS_BASE}/opt/vc/include/interface/vcos"
-	CFLAGS+=" -I${XBPS_CROSS_BASE}/opt/vc/include/interface/vmcs_host/linux"
-	CFLAGS+=" -I${XBPS_CROSS_BASE}/opt/vc/include/interface/vcos/pthreads"
-
-	LDFLAGS="-L${XBPS_CROSS_BASE}/opt/vc/lib -Wl,-rpath=/opt/vc/lib -lbcm_host -lvcos -lvchiq_arm"
 fi
 
 pre_configure() {
 	NOCONFIGURE=1 ./bootstrap
 
-	sed -i -e 's;tar cvvzf;tar cvzf;' share/Makefile.am share/Makefile.in
-	sed 's|pl_shader_alloc(tc->pl_ctx, NULL, 0, 0);|pl_shader_alloc(tc->pl_ctx, NULL, 0);|' -i modules/video_output/opengl/vout_helper.c
+	vsed -i -e 's;tar cvvzf;tar cvzf;' share/Makefile.am share/Makefile.in
 
 }
 
@@ -98,7 +90,7 @@ pre_install() {
 		# install .lua files instead of precompiled .luac ones
 		# scripts precompiled with 64 bit host compiler won’t run on 32 bit target
 		# same with respect to endianess
-		sed -i -e "s/\(@list='\$(nobase_vlclib_DATA\)/\1:.luac=.lua/" share/Makefile
+		vsed -i -e "s/\(@list='\$(nobase_vlclib_DATA\)/\1:.luac=.lua/" share/Makefile
 	fi
 }
 
```

Apply and push this diff and remove the common/shlibs change.

Reply:
fixed in 5b9db217